16-slices/06-slice-expressions/1-theory/main.go

```diff
@@ -35,5 +35,5 @@ func main() {
 	s.Show("msg[1:5]", msg[1:5])
 	s.Show("msg[1:]", msg[1:])
 
-	s.Show("append(msg)", append(msg[:4], '!'))
+	s.Show("append(msg)", append(msg[:2], 'y', '!'))
```

Comment:
I show this code in the lecture video, so we shouldn't change it here, can you remove this commit?
@firasuke

Reply:
Done!